### PR TITLE
Aragon Monthly 6th Edition

### LINKED ISSUE #172: Editor: Compile Issue 06 of Aragon Monthly
# Editor: Compile Issue 06 of Aragon Monthly

This issue is for a community contributor to compile the whole issue of Aragon Monthly Issue 06 that encompasses the content from June of 2018 in MarkDown files in the given format and structure.

The issue has to be closed in a single Pull Request that has all the necessary files and additions incorporated into them as defined below.

## News
- `docs/news/index.md` has to have all of the Issues from https://github.com/aragon/aragon-monthly/issues titled *Issue 06: add ... to News*
  - List of all items to be included can also be found at https://github.com/aragon/aragon-monthly/projects/6
- All the items from `news/index.md` have to be also included in the `news/tag.md` respective files based on the tag given to them in `news/index.md`. Ie. Aragon related news have to be included in `news/aragon.md`
- All news items in `news/index.md` and `tags.md` should be in chronological order based on their date of publishing indicated in the original news item from oldest to newest

- Examples of proper formatting and structure for news:
  - https://github.com/aragon/aragon-monthly/blob/master/docs/archive/issue02/news/index.md
  - https://raw.githubusercontent.com/aragon/aragon-monthly/master/docs/archive/issue02/news/index.md
  - https://github.com/aragon/aragon-monthly/blob/master/docs/archive/issue02/news/daos.md
  - https://raw.githubusercontent.com/aragon/aragon-monthly/master/docs/archive/issue02/news/daos.md

_Featured news on top of the pages are optional, the Monthly staff can choose and add these as well_ 
Guide for news:
- https://monthly.aragon.one/guides/guide_for_submitting_news/

## Articles

Any Articles submitted during the period for a given Issue of Monthly need to be added to `articles/index.md` under their respective Categories as well as to the `articles/category.md` files in a similar fashion as News items.
  - This includes any possible Columns as well
  - List of all items to be included can also be found at https://github.com/aragon/aragon-monthly/projects/6

- Examples of proper formatting and structure for articles:
  - https://github.com/aragon/aragon-monthly/blob/master/docs/archive/issue01/articles/index.md
  - https://raw.githubusercontent.com/aragon/aragon-monthly/master/docs/archive/issue01/articles/index.md
  - https://github.com/aragon/aragon-monthly/blob/master/docs/archive/issue01/articles/opinion/thoughts_on_governance_network_effects.md
  - https://raw.githubusercontent.com/aragon/aragon-monthly/master/docs/archive/issue01/articles/opinion/thoughts_on_governance_network_effects.md
  - https://github.com/aragon/aragon-monthly/blob/master/docs/archive/issue01/articles/opinion/how_aragon_create_a_fairer_world.md
  - https://raw.githubusercontent.com/aragon/aragon-monthly/master/docs/archive/issue01/articles/opinion/how_aragon_create_a_fairer_world.md

Guide for articles:
- https://monthly.aragon.one/guides/guide_for_submitting_articles/

## Classifieds

Any Classifieds submitted during the period for a given Issue of Monthly need to be added to `classifieds/index.md`.

- Examples of proper formatting and structure for classifieds:
  - https://github.com/aragon/aragon-monthly/blob/master/docs/archive/issue02/classifieds/index.md
  - https://raw.githubusercontent.com/aragon/aragon-monthly/master/docs/archive/issue02/classifieds/index.md

Guide for classifieds:
- https://monthly.aragon.one/guides/guide_for_submitting_classifieds/

___
**Examples of what a proper, successful Pull Request for this issue would look like:**
- https://github.com/aragon/aragon-monthly/pull/138
  - https://github.com/dxiri/aragon-monthly/blob/76ae0b8008a2f0e0e2f05323ba37b77965c15024/docs/news/index.md
  - https://github.com/aragon/aragon-monthly/pull/138/files#diff-3c9fd7cf9b88131ef6b326249549ba9d
  - https://github.com/dxiri/aragon-monthly/blob/76ae0b8008a2f0e0e2f05323ba37b77965c15024/docs/news/daos.md
  - https://github.com/aragon/aragon-monthly/pull/138/files#diff-c80ab4a8e4a41b38c9f51d2eb0bf5588

- https://github.com/aragon/aragon-monthly/pull/145
  - https://github.com/aragon/aragon-monthly/pull/145/files
___
## Notes

We will allow for a **maximum of 3 revisions** to the Pull Request based on feedback given by the Monthly staff, if the PR still doesn't meet the quality and standard, we will close the PR.

This Issue **should be finished and merged as soon as possible**, this is first-come, first-served; the first proper Pull Request will be merged and will be awarded the associated bounty.

_This task and bounty will be recurring, it will be repeated every month, if we see a contributor repeatedly successfully accomplish the task, the person can be added to the Monthly editor staff and they will be able to assign themselves and will have the ability to claim the work and associated bounty in the future._

### PR DESCRIPTION
**Pull request by tao**
# Compile Aragon Monthly

### Closing of a referenced Issue
Closes #172 

### Content category
_Article_ / _Column_ / _News_ / _Classified_

#### Content type:
_Editorial_

### Title
Aragon Monthly 6th Edition

### Description
Contains all the news for Issue 06, and one classifieds article. As far as I understand there are no Articles this month? I addd some suitable images for missing images themselves, and wrote custom excerpt for each article. 

Issue05 was archived and linked in the mkdocs.yml file.

On the font page I've taken the 6 most recent compelling articles, but I'm happy to switch them out if want to feature different articles. 

Let me know if you have any changes or would like to feature any articles this month.

### CODE REVIEW

**Comment by tao** [comment]
The custom images I used in the articles are:

[Britannica](https://www.britannica.com/topic/New-York-Central-Railroad-Company) (As far as I know there's no copyright issue with using this image from Britannica.)
- **Crypto Commons**

[Unsplashed](https://unsplash.com)
- **What is on-chain cryptocurrency governance? Is it plutocratic?**
- **Towards An International Court of Smart Contract Arbitration**
- **Ethereum’s Forgotten Treasure: DAOs**

The other images are from the articles or links themselves.

**Comment by Smokyish** [comment]
@tao Thank you for the submission! There are 3 Articles under Opinion pieces that have been merged into `master` for Issue 06.

Will start a review of the PR and if you can in the mean time add the articles, that's great, if not, that's not a blocking issue.

**Comment by tao** [comment]
Right, sorry about that - I removed them because I believed they were from issue05. I've added them back in and done some editing, and added them to the front-page `index.md` and `mkdocs.yml` file too.

Some issues with the opinion pieces that I'm not sure how to deal with are:

**The importance of Governance:** 
The image mentioned in the markdown is missing, I'm not sure where to find that... its looking for (images/dao.jpg)

**Ethereum’s response to EIP0 Results: Governance & Community**
The first heading has code comments: "**// The first...**" That's how it looks in the medium article but we may want to remove it so its just: "**The first...**"

Are there any other issues remaining?

File: docs/news/aragon.md [hunk 1]
```diff
@@ -1,53 +1,20 @@
...
-Author [Tatu Kärki](https://blog.aragon.one/@Smokyish) on May 31 |  |  |
-**First Aragon app launched on Ethereum Mainnet for community sentiment signaling**. Today we are releasing an entirely new step for community members to signal their sentiment on the project and igniting the next level of participation in the project for all ANT holders. Survey app is the fourth app developed by our team in addition to the Token Manager, Voting & Finance apps seen previously in Aragon Core v0.5. And the first one to launch on the Ethereum Mainnet. |  |  |
-[Read More](https://blog.aragon.one/aragon-launches-survey-app-on-mainnet-ed5eefeb66f5) | [Read More](../news) | [Submit News!](../guides/guide_for_submitting_news.md) |
+[_Aragon_](aragon.md) | | |
+[<img src="https://cdn-images-1.medium.com/max/1000/1*-hd2CC4ZPZ6N5wmhH7YTZA.jpeg">](https://blog.aragon.one/aragon-core-v0-5-1-post-mortem-part-1-85b0897b9157) | | |
```
Comment (Smokyish) [inline docs/news/aragon.md]: Last, empty, columns could use the `../../images/monthly_no_image.png` image

File: docs/news/daos.md [hunk 2]
```diff
@@ -1,21 +1,11 @@
...
-|I’d experienced the joy of connecting without money — sharing meals and homes with complete strangers on Couchsurfing.com, gifting access to my car while I traveled, and experimenting with random acts of generosity. These experiences were so transformational that I volunteered to help shoot a documentary, One Couch at a Time, which highlighted the relational aspects of the burgeoning sharing economy.|You may have not heard of the abbreviation DAO, but it is a term that should keep popping up with more frequency during the blockchain boom. It stands for decentralized autonomous organizations. As companies are betting on and adopting blockchain, many believing that blockchain is the future, DAO could start playing an even bigger role.||
-|[Read More](https://hackernoon.com/should-couchsurfing-be-a-dao-6507646e34ef)|[Read More](https://www.inc.com/darren-heitner/never-heard-of-a-dao-heres-why-its-going-to-become-hottest-new-type-of-corporation.html)||
+[**How Daox Works — Part 1**](https://medium.com/daox/how-daox-works-part-1-a1d2a456cbe7) | [**Ethereum’s Forgotten Treasure: DAOs**](https://cryptoslate.com/ethereums-forgotten-treasure-daos/) |
```
Comment (Smokyish) [inline docs/news/daos.md]: Missing the third, empty, column.

File: docs/news/governance.md [hunk 3]
```diff
@@ -1,27 +1,35 @@
...
+[Read More](https://marginalrevolution.com/marginalrevolution/2018/06/towards-international-court-smart-contract-arbitration.html) | [Read More](https://blog.foam.space/exploring-ethereum-storage-costs-with-the-foam-developer-tools-96d84e1a06b5) | [Read More](https://medium.com/@richardred/what-is-on-chain-cryptocurrency-governance-is-it-plutocratic-bfb407ef6f1) |
+
+[**Voting and Governance in Security Tokens**](https://medium.com/coinmonks/voting-and-governance-in-security-tokens-1e3d041dabb8) | [**Crypto Commons — from tragedy to wealth of the commons**](https://blog.usejournal.com/crypto-commons-da602fb98138) | |
+:-----------:|:-----------:|:-----------:|
+[_Governance_](governance.md) | [_Governance_](governance.md) | |
+[<img src="https://cdn-images-1.medium.com/max/800/1*F7oHt_loAWz4EACe_SqRgQ.jpeg">](https://medium.com/coinmonks/voting-and-governance-in-security-tokens-1e3d041dabb8) | [<img src="https://cdn.britannica.com/668x448/09/134309-004-ABF9FE28.jpg">](https://blog.usejournal.com/crypto-commons-da602fb98138) | |
```
Comment (Smokyish) [inline docs/news/governance.md]: Last, empty, column could use the `../../images/monthly_no_image.png` image and the `Submit News!` link.

File: docs/news/index.md [hunk 4]
```diff
@@ -1,69 +1,51 @@
...
-| [Read More](https://blog.aragon.one/aragon-network-on-a-path-towards-a-digital-jurisdiction-752ccf4b9d5c) |[Read More](https://medium.com/betoken/we-need-to-formalize-governance-research-using-daos-e9393da427f3)| [Read More](https://blog.aragon.one/aragon-launches-survey-app-on-mainnet-ed5eefeb66f5) |
+[_Governance_](governance.md) | [_Governance_](governance.md) | |
+[<img src="https://cdn-images-1.medium.com/max/800/1*F7oHt_loAWz4EACe_SqRgQ.jpeg">](https://medium.com/coinmonks/voting-and-governance-in-security-tokens-1e3d041dabb8) | [<img src="https://cdn.britannica.com/668x448/09/134309-004-ABF9FE28.jpg">](https://blog.usejournal.com/crypto-commons-da602fb98138) | |
```
Comment (Smokyish) [inline docs/news/index.md]: Last, empty, column could use the `../../images/monthly_no_image.png` image and the `Submit News!` link.

**Comment by Smokyish** [comment]
@tao Great work, will merge this as is, please see our [How to claim an Aragon bounty using Status Open Bounty](https://wiki.aragon.one/tutorials/Status_Open_Bounty_Guide/) for claiming the bounty, payouts are done weekly so you will receive the payout within a week.

**Pull request merged**